lms/djangoapps/course_goals/models.py

```diff
@@ -34,3 +38,17 @@ def __unicode__(self):
 
     class Meta:
         unique_together = ("user", "course_key")
+
+
+@receiver(models.signals.post_save, sender=CourseEnrollment, dispatch_uid="create_course_goal_on_enroll")
+def set_course_goal_verified(sender, instance, **kwargs):  # pylint: disable=unused-argument, invalid-name
+    """Set the course goal to certify when the user enrolls as a verified user. """
+    ## If the user enrolls in a verified state or upgrades, create a course goal
+    if instance.mode == 'verified':
+        add_course_goal(instance.user, instance.course_id, 'certify')
```

Comment:
Same here.  Can we use GOAL_KEY_CHOICES.certify rather than `certify`?  Basically, looking to avoid sprinkling strings about as constants where possible.  If there is a reason why using choices is bad, maybe we should revive your option enum?

Reply:
?